app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaScreen.kt

```diff
@@ -646,7 +734,49 @@ fun AgendaScreenPreview_NightMode_NO() {
 }
 
 
-
+//            is AgendaEvent.ShowLogoutDialog -> {
+//                val dialogState = remember { mutableStateOf(true) }
+//
+//                if (dialogState.value) {
+//                    AlertDialog(
+//                        onDismissRequest = { dialogState.value = false },
+//                        title = {
+//                            Text(
+//                                text = stringResource(id = R.string.logout),
+//                                fontWeight = FontWeight.Bold,
+//                            )
+//                        },
+//                        text = {
+//                            Text(
+//                                text = stringResource(id = R.string.logout_confirmation),
+//                            )
+//                        },
+//                        confirmButton = {
+//                            TextButton(
+//                                onClick = {
+//                                    dialogState.value = false
+//                                    onAction(AgendaEvent.Logout)
+//                                }
+//                            ) {
+//                                Text(
+//                                    text = stringResource(id = R.string.logout),
+//                                    fontWeight = FontWeight.Bold,
+//                                )
+//                            }
+//                        },
+//                        dismissButton = {
+//                            TextButton(
+//                                onClick = { dialogState.value = false }
+//                            ) {
+//                                Text(
+//                                    text = stringResource(id = R.string.cancel),
+//                                    fontWeight = FontWeight.Bold,
+//                                )
+//                            }
+//                        },
+//                    )
+//                }
+//            }
```

Comment:
I'm personally not a friend of including commented out code in commits, since that doesn't add any value and just makes them larger. Deleting this code is fine - if you'll ever need it again Git got your back.

Reply:
OK, i will remove code like this from future PR's

Reply:
Is there an easy way to go back to this code, or do you have to look through each PR and Commit? (thats the only way I know how to do it, other than stashing)...

Reply:
I personally just use the Git commit history in Android Studio and go back to a commit where I know this code was contained in. You can then just right click and compare with local where you can easily include this code with a click on that little diff arrow in the commit-difference tab.

Works well if you work alone on the project. If others are also working on it, saving snippets sounds like a good approach, since then code gets easier lost and you could also better share these snippets with co-workers (since you might not even know there is a snippet)